zephyr/CMakeLists.txt

```diff
@@ -188,6 +189,7 @@ if (CONFIG_SOC_SERIES_INTEL_CAVS_V18)
 	zephyr_library_sources_ifdef(CONFIG_INTEL_DMIC
 		${SOF_DRIVERS_PATH}/intel/dmic/dmic.c
 		${SOF_DRIVERS_PATH}/intel/dmic/dmic_computed.c
+		${SOF_DRIVERS_PATH}/intel/dmic/dmic_nhlt.c
```

Comment:
I think I should add here ifdefs for CONFIG_INTEL_DMIC_TPLG_PARAMS and CONFIG_INTEL_DMIC_NHLT. There's no more #ifdefs there for whole modules.

Reply:
Done